sources/packages/web/src/types/contracts/DesignationAgreementLocationStatusContract.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Possible status for a designation agreement location.
+ */
+export enum DesignationAgreementLocationStatus {
```

Comment:
I would recommend renaming the enum to DesignationApprovalStatus or DesignatedStatus or something event better. Same mentioned on API side.